.vscode/launch.json

```diff
@@ -0,0 +1,40 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Verify",
+      "type": "go",
+      "request": "launch",
+      "mode": "auto",
+      "program": "${workspaceFolder}/cmd/ratify",
+      "args": [
+        "verify",
+        "-s",
+        "${input:subject}"
+      ],
+    },
+    {
+      "name": "Serve",
+      "type": "go",
+      "request": "launch",
+      "mode": "auto",
+      "program": "${workspaceFolder}/cmd/ratify",
+      "args": [
+        "serve",
+        "--http",
+        ":6001"
+      ],
```

Comment:
Should the crd manager be enabled "--enable-crd-manager" to mirror what is deployed in k8s ? https://github.com/deislabs/ratify/blob/main/charts/ratify/templates/deployment.yaml#L44

Reply:
Not sure! The team and I have been using this for development, and we've been updating `config.json` during dev

I think it should probably stay omitted. In a VS Code launch, you're on a local machine / in a devcontainer instead of a k8s cluster. Will verify, but I expect adding that flag will cause the launch to fail since Ratify can't talk to the k8s apiserver

Reply:
got it, thanks for clarification Noel